docs/general/dns-providers.md

```diff
@@ -1041,6 +1041,15 @@ Family mode offers protection by "Safe" resolvers and blocks adult sites.
 
 [ControlD](https://controld.com/free-dns) is public DNS service which provides ads, malware, and social media blocking.
```

Comment:
you need to add a header above like it's done for other DNS providers: https://adguard-dns.io/kb/general/dns-providers/

Reply:
```suggestion
[ControlD](https://controld.com/free-dns) is a customizable DNS service with proxy capabilities. This means it not only blocks things (ads, porn, etc.), but can also unblock websites and services.
```